src/components/new-request/search.vue

```diff
@@ -14,7 +14,7 @@
       <!-- New BC Business flow -->
       <template v-if="isNewBcBusiness">
         <EntityType v-if="getLocation" />
-        <CompanyType v-if="entity_type_cd" />
+        <CompanyType v-if="getEntityTypeCd && isNumberedEntityType" />
```

Comment:
You probably need to add a check for Societies, since it's conditional on a FF.

Reply:
Sev, in my last commit, I fixed all the issues resulting from conditionals by adding them to the template.

I don't know if it's kind of ugly, but I don't think there's escaping them. Please let me know what you think. Now, all business are looking exactly like dev.

Reply:
You have very little ugliness in the template 👍 

I think the logic in the 3 global button getters is fine.

However, I think you still need to handle Societies.

Reply:
Ahh, I missed that, my bad. There's the `isSocietyEnabled` FF. I'm on it.

Reply:
I am yet to fix the society dialog when FF is false. The code is written in a very weird way and I'm trying to figure that out.

Reply:
Never mind, it works totally fine right now. If the FF is disabled, the dialog will show. I tried that by removing the ! on line 255 in `pick-entity-or-conversion.vue` https://github.com/JazzarKarim/namerequest/blob/ce013f6d0ad827699da28c3a6374d2c7416bc0de/src/components/dialogs/pick-entity-or-conversion.vue#L255

So, societies should be OK after latest commit.